gsa/src/web/components/dashboard/dashboard.js

```diff
@@ -90,39 +94,31 @@ const RowPlaceHolder = styled.div`
   margin: 15px 0;
 `;
 
-export class Dashboard extends React.Component {
-  constructor(...args) {
-    super(...args);
-
-    const {permittedDisplays = []} = this.props;
-
-    this.components = {};
-    permittedDisplays.forEach(displayId => {
-      const display = getDisplay(displayId);
-
-      if (isDefined(display)) {
-        this.components[displayId] = display.component;
-      } else {
-        log.warn('Unknown Dashboard display', displayId);
-      }
-    });
-
-    this.handleItemsChange = this.handleItemsChange.bind(this);
-    this.handleRowResize = this.handleRowResize.bind(this);
-    this.handleUpdateDisplay = this.handleUpdateDisplay.bind(this);
-    this.handleRemoveDisplay = this.handleRemoveDisplay.bind(this);
-
-    this.getDisplaysById = memoize(rows => getDisplaysById(rows));
-  }
+export const Dashboard = props => {
```

Comment:
I would love if we can destruct all used props already here in the Dashboard function definition.

```javascript
const Dashboard = ({
    permittedDisplays = [],
    defaultDisplays,
   maxItemsPerRow = DEFAULT_MAX_ITEMS_PER_ROW,
   ...
}) => {
```

but i guess it is something for another pr.